src/shared/ui/date-lang/date-lang.tsx

```diff
@@ -1,31 +1,36 @@
-/* eslint-disable @stylistic/jsx-one-expression-per-line */
+import classNames from 'classnames/bind';
+import { dayJS } from '@/app/services/dayjs';
 import micIcon from '@/shared/assets/icons/mic.svg';
 import noteIcon from '@/shared/assets/icons/note-icon.svg';
 import Image from '@/shared/ui/image';
 
 import styles from './date-lang.module.scss';
 
+const cx = classNames.bind(styles);
+
 interface DateLangProps {
   startDate: string;
   mode: string;
   language: string[];
-  type?: string;
+  withMargin?: boolean;
 }
 
-export const DateLang = ({ startDate, language, mode, type = '' }: DateLangProps) => {
-  const splittedLanguage = language.slice().join('/');
+export const DateLang = ({ startDate, language, mode, withMargin }: DateLangProps) => {
+  const dateAttr = dayJS(startDate).format('YYYY-MM-DD');
 
   return (
-    <div className={`${styles.info} ${styles[type]}`}>
-      <p className={styles.date}>
-        <Image className={styles.note} src={noteIcon} alt="note-icon" lazy="false" />
-        <span>Start date {startDate ?? 'Not set'}</span>
+    <section className={cx('info', { margin: withMargin })}>
+      <p className={cx('date')}>
+        <Image className={cx('icon')} src={noteIcon} alt="note-icon" lazy="false" />
+        <span>Start:</span>
+        <time dateTime={dateAttr}>{startDate}</time>
       </p>
-      <p className={styles.lang}>
-        <Image className={styles.microphone} src={micIcon} alt="microphone icon" lazy="false" />
-        <span className={styles.language}>{splittedLanguage}</span>
-        <span className={styles.mode}> • {mode}</span>
+      <p className={cx('additional-info')}>
```

Comment:
Please remove this from the component as we agreed.

Reply:
This component reused in course-card now, no need duplicate code

Reply:
As we decided earlier, we rename `DateLang` to `Date` and remove the lang-info from this component